lib/cc/services/github_pull_requests.rb

```diff
@@ -73,4 +89,82 @@ def response_includes_repo_scope?(response)
   def test_status_code
     422
   end
+
+  def welcome_comment_implemented?
+    true
+  end
+
+  def should_post_welcome_comment?
+    config.welcome_comment_enabled && @payload.fetch("first_contribution", false)
+  end
+
+  ALREADY_COMMENTED_PATTERN = /Since this is the first PR/
```

Comment:
I think this then falls under YAGNI. That hypothetical third component doesn't exist, and if it ever does I think it'll mean we need to consider some deeper refactoring here.